coops-ui/src/router.ts

```diff
@@ -1,10 +1,10 @@
 import Vue from 'vue'
-import Router from 'vue-router'
+import VueRouter from 'vue-router'
```

Comment:
What is the benefit/difference between Router and VueRouter?

Reply:
VueRouter is the current way to import it.
I think "Router" was the old way, but it resolves to VueRouter anyway.